data/selinux/snappy.te

```diff
@@ -633,7 +663,9 @@ allow init_t snappy_unconfined_snap_t:process { sigkill signull signal };
 
 # snap tools can be invoked by the regular user, make sure that things get
 # proper labels
-gen_require(` type unconfined_t; ')
```

Comment:
I find all the changes above noisy, there seem unrelated to the actual change below, am I correct?

Reply:
It's in a separate patch though. FWIW, we should have used the format from the start